webapp/src/main/java/org/apache/atlas/web/rest/MigrationREST.java

```diff
@@ -0,0 +1,154 @@
+package org.apache.atlas.web.rest;
+
+import javax.ws.rs.Path;
+import org.apache.atlas.RequestContext;
+import org.apache.atlas.annotation.Timed;
+import org.apache.atlas.exception.AtlasBaseException;
+import org.apache.atlas.model.instance.AtlasEntity;
+import org.apache.atlas.model.instance.EntityMutationResponse;
+import org.apache.atlas.repository.store.graph.AtlasEntityStore;
+import org.apache.atlas.repository.store.graph.v2.AtlasEntityStream;
+import org.apache.atlas.repository.store.graph.v2.EntityStream;
+import org.apache.atlas.repository.store.users.KeycloakStore;
+import org.apache.atlas.transformer.PreProcessorPoliciesTransformer;
+import org.apache.atlas.utils.AtlasPerfTracer;
+import org.apache.atlas.web.util.Servlets;
+import org.apache.commons.collections.CollectionUtils;
+import org.apache.commons.lang.StringUtils;
+import org.keycloak.representations.idm.RoleRepresentation;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.stereotype.Service;
+
+import javax.inject.Inject;
+import javax.inject.Singleton;
+import javax.ws.rs.*;
+import javax.ws.rs.core.MediaType;
+import java.util.ArrayList;
+import java.util.List;
+
+import static org.apache.atlas.authorize.AtlasAuthorizerFactory.ATLAS_AUTHORIZER_IMPL;
+import static org.apache.atlas.authorize.AtlasAuthorizerFactory.CURRENT_AUTHORIZER_IMPL;
+import static org.apache.atlas.repository.Constants.*;
+
+@Path("migration")
```

Comment:
if this is temp you can ignore. Else please include the usecase in the API for which the migraiton is done.

Reply:
We will need to migrate Connections/collections/Persona/Purpose/API token
These are migration helper APIs for Connections & collections
Both will create necessary bootstrapped policies & create required roles on Keycloak